Makefile

```diff
@@ -21,27 +21,27 @@ help:
 	@echo "build-base    - (re)build base docker image"
 	@echo "docs          - generate Sphinx HTML documentation, including API docs"
 
-.docker-build-base:
-	${MAKE} build-base
+.env:
+	@if [ ! -f my.env ]; then \
+		echo "Copying .env-dist to .env..."; \
+		cp .env-dist .env; \
+	fi
 
 .docker-build:
 	${MAKE} build
 
-build-base:
-	${DOCKERCOMPOSE} -f docker-compose.build.yml build --pull base
-	-rm -f .docker-build
-	touch .docker-build-base
-
-build: .docker-build-base
-	${DOCKERCOMPOSE} -f docker-compose.build.yml build web
+build:
+	${DOCKERCOMPOSE} build web
+	${DOCKERCOMPOSE} build prod
+	${DOCKERCOMPOSE} build app
 	touch .docker-build
 
 rebuild: clean .docker-build
 
-run: .docker-build
+run: .docker-build .env
 	${DOCKERCOMPOSE} up web
```

Comment:
I'd probably change this to:

```
${DOCKERCOMPOSE} up web app
```

Since the `web` container is proxying the `app` container, you wouldn't see the Django logging output and that makes it harder to do work in Django land.